tests/run_tests.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+echo "Starting database"
+
+# initialize database
+initdb --username=postgres
+
+pg_ctl -o "-c listen_addresses='127.0.0.1'" -w start
+
+psql -v ON_ERROR_STOP=1 --username postgres --no-password --dbname postgres <<-'EOSQL'
+SET TIME ZONE 'UTC';
+CREATE DATABASE lega;
+EOSQL
+
+for f in /docker-entrypoint-initdb.d/*
+do
+	echo "$0: running $f";
+	echo
+	psql -v ON_ERROR_STOP=1 --username postgres --no-password --dbname lega -f "$f";
+	echo
+done
```

Comment:
some scripts depend on the `local_ega` schema so they throw an error but I guess this will be remedied in coming prs.

Reply:
yeah, removed the old schemas - they'll be replaced in a different PR, but maybe it's cleaner to remove them here already :)